### ISSUE TITLE
Error with variant of untagged enum RequestUnion with GraphQL collections

### ISSUE THREAD

**Issue by soundstep** [kevinswiber/postman2openapi#41]
Hi,

I'm getting this error:
```
data did not match any variant of untagged enum RequestUnion at line 30 column 4
```

With:
```
postman2openapi "Pact bi-directional.postman_collection.json" > result.yml
```

You can find the postman collection there:
https://gist.github.com/soundstep/aba44f8b453e28418814122209c176db

- `POST discovery PRD`: GraphQL POST request using the `graphql` body field in postman
- `POST discovery PRD raw`: GraphQL POST request using the `raw` body field in postman
- `GET discovery PRD`: GraphQL GET request using a query string

It seems to be happening when using the graphql body option in postman.

<img width="1298" alt="Screenshot 2022-08-01 at 15 17 46" src="https://user-images.githubusercontent.com/133293/182168575-2f36f8a4-b007-42b5-a0c4-96d52c85c3c8.png">

Can you advice?

**Comment by kevinswiber** [comment]
Thanks for the draft PR!  I'll take a look later today.

**Comment by kevinswiber** [comment]
Fixed by #42.

This will get better as we figure out merging conflicting operations and examples.
